### ISSUE TITLE
task execution branch: seek coord until then

### ISSUE THREAD

**Issue by phrokton** [ermiyaeskandary/Slither.io-bot#306]
In the task execution branch

Would it be better to have :

_Seek coord, until food density greater than, resume when food density less than_

Additionally in seek coord mode I have watched snake plough through piles of food ignoring the bounty.

**Comment by clemens-tolboom** [comment]
@phrokton your description is a little fuzzy. Why should snake go to (x,y) until food density is enough?

I would opt for 'Goto food density level X' which is when not found nearby either inward or outward.

**Comment by phrokton** [comment]
> 'Goto food density level X'

Yes, @clemens-tolboom , I would much prefer a head towards [center to find] better food but the fork already has a MoveToXY.  Better food can be defined in SnakeParamsBySize.

I watched several instances fritter about on the periphery.  Returning hours later than game was still in play and the bot had not grown much more with sparse edge food.

There is not a priority level setting for MoveToXY.  But for seek 'density level X' bot ought not fly past a juicy carcass.
